src/Transport/Utils/CircularBuffer.cs

```diff
@@ -250,6 +250,8 @@ public void CopyTo(T[] array)
 
         void CopyTo(int index, T[] array, int arrayIndex, int count)
         {
+            _ = index;
```

Comment:
Could we just remove the parameter instead?

Reply:
Similar to https://github.com/Particular/NServiceBus.AzureServiceBus/pull/942#discussion_r557686891